cli/src/main/scala/com/codacy/analysis/cli/command/CLIApp.scala

```diff
@@ -144,6 +144,8 @@ final case class Analyze(
   commitUuid: Option[Commit.Uuid] = Option.empty,
   @ExtraName("s") @ValueDescription("[default: false] - Force using a commit UUID")
   skipCommitUuidValidation: Int @@ Counter = Tag.of(0),
+  @ExtraName("g") @ValueDescription("[default: false] - Skip the check for uncommitted files in the analysis directory")
```

Comment:
Why `-g`?
I think you can avoid defining an `ExtraName` altogether and just allow the long option.
`skip-uncommitted-files-check` has not `g`ees :-D
```suggestion
  @ValueDescription("[default: false] - Skip the check for uncommitted files in the analysis directory")
```